frontend-react/src/utils/ScreenWidth.ts

```diff
@@ -0,0 +1,11 @@
+export default function isScreenWidth(size: "mobile" | "tablet" | "desktop") {
```

Comment:
Would you mind adding tests for this?

Could be as easy as something like:
```js
import isScreenWidth, { AvailableScreenWidths } from "./ScreenWidth";

// UNTESTED
describe("isScreenWidth", () => {
    function setWindowWidth(width: number) {
        // I think innerWidth is a read-only property so we can get around it this way 🤷 
        Object.defineProperty(window, "innerWidth", {
            configurable: true,
            writable: true,
            value: width,
        });
    }
    const originalWindowWidth = window.innerWidth;

    afterAll(() => {
        setWindowWidth(originalWindowWidth);
    });

    describe("when on a mobile viewport", () => {
        beforeEach(() => {
            setWindowWidth(480);
        });

        test("it only returns true for 'mobile'", () => {
            expect(isScreenWidth("mobile")).toEqual(true);
            expect(isScreenWidth("tablet")).toEqual(false);
            expect(isScreenWidth("desktop")).toEqual(false);
        });
    });

   describe("when on a tablet viewport", () => {
        beforeEach(() => {
            setWindowWidth(768);
        });

         test("it only returns true for 'tablet'", () => { ... });
   });

   // ...and so on
});

```

Reply:
Sounds totally reasonable!